nemo_text_processing/text_normalization/verbalizers/money.py

```diff
@@ -34,8 +40,17 @@ class MoneyFst(GraphFst):
         decimal: DecimalFst
     """
 
-    def __init__(self, decimal: GraphFst):
+    def __init__(self, decimal: GraphFst, deterministic=True):
         super().__init__(name="money", kind="verbalize")
+
+        def _get_minor_currencies(file):
+            minor_currencies = []
+            with open(get_abs_path(file), 'r') as f:
+                for line in f:
+                    min_cur = line.strip()
+                    minor_currencies.append(pynini.closure(pynutil.insert(min_cur), 0, 1))
```

Comment:
can you use string_file() instead of this helper function to create graph?

Reply:
it does, if file has only 1 column its creating acceptor

Reply:
I don't need an accepter, I need to insert a new string. we're keeping track of major currencies only, this is a way to add minor currencies, like cents